BigAssApi.js

```diff
@@ -60,6 +60,26 @@ function FanMaster (numberOfExpectedFans) {
             this.rescanForFans();
         }
     }.bind(this)
+    */
+        handleNewFan = function(msg, address) {
+        if (msg[0] == "ALL") {
+            return; // Message not addressed to us
+        }
+        if (msg[4] == "FAN,LSERIES") { //this is less than perfect, but at least allows a program to continue passed the onfanfullyupdated function when a wall control is present - obviously needs to not be LSERIES specfic - need a function to just look at the "FAN" part of that string
```

Comment:
Hmm, for these messages - what's the full message that is found from the fan? Unfortunately I think this breaks non-LSeries fans. If we want to get just the FAN part we can use do:
```
var onlyFanPortion = split(msg[4], ",")[0] // Grab first part of string before ","
if (onlyFanPortion == "FAN") {
...
```